.github/ISSUE_TEMPLATE/V&V_request.md

```diff
@@ -69,6 +69,8 @@ Application:
 - [ ] Device enrolled in Hexnode
 - [ ] Equipment set up for V&V execution
 - [ ] Parse account created in correct server
+- [ ] Verify if V&V account should be an internal or external parse account
+- [ ] Set expiration date to parse account
```

Comment:
Couple things:

1. `Parse` is capitalized
2. Upon completion of the V&V, set the expiration date of the Parse account used during the V&V to the current date and time.